### PR TITLE
Update codeowners

### PR DESCRIPTION
**Pull request by adamschmideg**
I updated codeowners based on the activity of the last half year. Please, feel free to make changes.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,15 +4,21 @@
 accounts/usbwallet              @karalabe
 accounts/scwallet               @gballet
 accounts/abi                    @gballet
+cmd/clef                        @holiman
+cmd/puppeth                     @karalabe
 consensus                       @karalabe
-core/                           @karalabe @holiman
-eth/                            @karalabe
+consensus/clique                @holiman
```
Comment (fjl) [inline .github/CODEOWNERS]: `@karalabe` should be added here

Comment (adamschmideg) [reply .github/CODEOWNERS]: Done

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -4,15 +4,21 @@
 accounts/usbwallet              @karalabe
 accounts/scwallet               @gballet
 accounts/abi                    @gballet
+cmd/clef                        @holiman
+cmd/puppeth                     @karalabe
 consensus                       @karalabe
-core/                           @karalabe @holiman
-eth/                            @karalabe
+consensus/clique                @holiman @karalabe
```
Comment (karalabe) [inline .github/CODEOWNERS]: Not sure I follow the addition of Martin here.

Comment (adamschmideg) [reply .github/CODEOWNERS]: My misunderstanding. Removed.

**Pull request merged**